docs/source/cloud-setup/cloud-permissions/gcp.rst

```diff
@@ -1,19 +1,56 @@
-.. _cloud-permissions-gcp:
-
 GCP
 =============
 
 
+.. _gcp-service-account:
```

Comment:
same as the comment above, we should have a sentence at the begining referring to normal installation and setup, and claiming that the following is optional